spec/models/user_spec.rb

```diff
@@ -232,4 +232,158 @@
       it { expect(subject.is_member_or_admin?).to be_truthy }
     end
   end
+
+  describe '#is_in_company_numbered?(company_num)' do
+
+    default_co_number = '5562728336'
+    describe 'not yet a member, so not in any full companies' do
+
+      describe 'user: no applications, so not in any companies' do
+        subject { create(:user, is_member: false) }
+        it { expect(subject.is_in_company_numbered?(default_co_number)).to be_falsey }
+      end
+
+      describe 'user: 1 saved application' do
+        subject { create(:user_with_membership_app) }
+        it { expect(subject.is_in_company_numbered?(default_co_number)).to be_falsey }
+      end
+
+      describe 'user: 2 application' do
+        subject { create(:user_with_2_membership_apps) }
+        it { expect(subject.is_in_company_numbered?(default_co_number)).to be_falsey }
+      end
+    end
+
+    describe 'is a member, so is in companies' do
+
+      describe 'member with 1 app' do
+        let(:member) { create(:member_with_membership_app) }
+        it { expect(member.is_in_company_numbered?(default_co_number)).to be_truthy }
+      end
+
+      describe 'member with 2 apps, both with same (1) company' do
+        let(:member) { m = create(:member_with_membership_app)
```

Comment:
use `{ }` when the code in the block can fit in a single line., otherwise use `do ... end`